opm/autodiff/FlowMainEbos.hpp

```diff
@@ -541,13 +568,51 @@ namespace Opm
         {
             bool output      = param_.getDefault("output", true);
             bool output_ecl  = param_.getDefault("output_ecl", true);
-            const Grid& grid = this->grid();
-            if( output && output_ecl && output_cout_)
+            if( output && output_ecl )
             {
-                const EclipseGrid& inputGrid = eclState().getInputGrid();
-                eclipse_writer_.reset(new EclipseWriter(eclState(), UgGridHelpers::createEclipseGrid( grid , inputGrid )));
-                eclipse_writer_->writeInitial(geoprops_->simProps(grid),
-                                              geoprops_->nonCartesianConnections());
+                const Grid& grid = this->globalGrid();
```

Comment:
I see. In the future, I should probably read the full diff before making any comments...